x-pack/plugins/apm/server/lib/services/get_services/get_services_items_stats.ts

```diff
@@ -195,7 +243,13 @@ export const getErrorRates = async ({
     const errorsPerMinute = bucket.doc_count / deltaAsMinutes;
```

Comment:
Perhaps to avoid confusion with the `dateBucket`
```suggestion
    const errorsPerMinute = serviceBucket.doc_count / deltaAsMinutes;
```